tensorflow/core/platform/BUILD

```diff
@@ -426,15 +426,6 @@ cc_library(
     ],
 )
 
-py_test(
-    name = "ram_file_system_test",
-    srcs = ["ram_file_system_test.py"],
```

Comment:
Should we also delete/move this file?

Reply:
Sorry if I am misunderstanding, but I believe `ram_file_system_test.py` _is_ being moved: https://github.com/tensorflow/tensorflow/commit/ee7a51028fb04dae61936e702449d493997741e8#diff-18772cb1894d1c5e6364f679cee08b916774e945094644a52b17735188b9f907

Reply:
Sorry, I had the page displaying the status wrong.